<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAddInput">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddWire">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Wire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddAND">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>AND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
